runtime/outer-PEC.js

```diff
@@ -79,16 +80,17 @@ class OuterPEC extends PEC {
     };
 
     this._apiPort.onArcCreateSlot = ({callback, arc, transformationParticle, transformationSlotName, hostedParticleName, hostedSlotName}) => {
+      let hostedSlotId;
       if (this.slotComposer) {
-        var hostedSlotId = this.slotComposer.createHostedSlot(transformationParticle, transformationSlotName, hostedParticleName, hostedSlotName);
+        hostedSlotId = this.slotComposer.createHostedSlot(transformationParticle, transformationSlotName, hostedParticleName, hostedSlotName);
       }
       this._apiPort.CreateSlotCallback({}, {callback, hostedSlotId});
     };
 
     this._apiPort.onArcLoadRecipe = async ({arc, recipe, callback}) => {
       let manifest = await Manifest.parse(recipe, {loader: this._arc._loader, fileName: ''});
       let error = undefined;
-      var recipe = manifest.recipes[0];
+      recipe = manifest.recipes[0];
```

Comment:
This is where @sjmiles made a Safari-focused fix, and will conflict -- resolve before landing?

Reply:
OK, rebased!